src/lib/db/server.js

```diff
@@ -19,7 +19,7 @@ export function createServer(serverConfig) {
 
     config: {
       ...serverConfig,
-      host: serverConfig.host || serverConfig.socketPath,
+      host: serverConfig._connectionMethod === 'default' ? serverConfig.host : serverConfig.socketPath,
```

Comment:
I don't think _connectionMethod is available here, I think this is the config provided by `connection_provider.js`